drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/main/java/org/drools/workbench/screens/testscenario/backend/server/ScenarioRunnerService.java

```diff
@@ -146,7 +143,7 @@ public void testAssumptionFailure(Failure failure) {
 
         testResultMessageEvent.fire(
                 new TestResultMessage(
-                        identity.getIdentifier(),
+                        TEST_SCENARIO_SERVICE_ID,
```

Comment:
I am not exactly sure about this. Since the `TestResultMessage` is directly in guvnor, it seems like something more generic, so not sure if removing the identity from there is alright.